src/state/actions/squadActions.js

```diff
@@ -0,0 +1,4 @@
+export const SET_SQUAD_FACEOFFS = 'SET_SQUAD_FACEOFFS';
```

Comment:
the faceoff action In our current repo was the exact same as this, not understanding the reasoning by adding this

Reply:
I didn't have it in my repo. I think I had to add it.